tock/projects/views.py

```diff
@@ -72,7 +71,7 @@ def project_timeline(project, period_limit=5):
                 break
             periods.append(report_date)
 
-        groups[tc.user][report_date] = float(t.hours_spent)
+        groups[tc.user.user_data.display_name][report_date] = float(t.hours_spent)
```

Comment:
as above, it *may* be better to use self.user.get_full_name() rather than traverse the DB to user_data

Reply:
hmmmm is putting this on `user_data` the wrong place? i guess i'm just concerned about those edge cases where there is no `full_name` and it'll just be blank.

Reply:
I think we can handle that on the data entry side and simply say all employees must have first and last names populated (they do)

Reply:
@Jkrzy is this where your select_related is helping us not have the extra query? I'm still feeling we should just use `user.get_full_name()` but if there's no DB hit it really doesn't matter.

Reply:
Yep! We stuck with handling cases where folks don't have a First/Last in the system

Reply:
Cool, cool. That shouldn't ever happen given how folks are onboarded, but if there's no DB hit, that's great.